addons/block_code/serialization/block_script_serialization.gd

```diff
@@ -28,6 +28,8 @@ const SCENE_PER_TYPE = {
 @export var generated_script: String
 @export var version: int
 
+var block_code_node: BlockCode
```

Comment:
A potential future improvement here would be to cache the parent properties when set so that `_get_parent_node_property_info` didn't have to iterate the entire property list every time. I don't think that's worth it right now, though.